pom.xml

```diff
@@ -1,38 +1,63 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
+
   <groupId>com.avides.springboot.testcontainer</groupId>
   <artifactId>springboot-testcontainer-common</artifactId>
+  <version>0.1.0-RC16</version>
```

Comment:
Warum nicht 1.0.0-RC1? So könnte man denken, dass die Version mit der 0.1.0 kompatibel ist

Reply:
Fixed in d03f569397641d5225bb4b53bec1f02c13e13bf3 and 3d5601d9d770f8d740939010b149391a161a6f86.